docs/howto/build-run.md

```diff
@@ -75,6 +75,8 @@ If you're starting with iOS development, be sure to [install
 CocoaPods](https://guides.cocoapods.org/using/getting-started.html),
 as the guide instructs, then rerun `yarn`.
 
+Start the Metro bundler server using `react-native start`.
+
```

Comment:
This is one of the steps of the upstream instructions:
https://reactnative.dev/docs/environment-setup
just before `react-native run-android` (or actually `npx react-native run-android`, in their version.) There are a lot of steps there, and I don't want to try to duplicate them.

It looks like that instruction is new in v0.62 -- it's not in the v0.61 docs:
https://reactnative.dev/docs/0.61/getting-started.html
So in short I think the audience that needs to be specifically told about this step is people who've *already* gone through these instructions before the v0.62 release. Like you :slightly_smiling_face: and potentially other existing contributors, but not new readers of these instructions.

Reply:
Makes sense.

Reply:
Deleted the commit.